client/mixins/apiRequestHelpers.js

```diff
@@ -0,0 +1,12 @@
+export default {
```

Comment:
I'm not married to this mixin existing, but I thought it could provide some utility beyond my changes. If you don't like it, just say the word and I'll nuke it + refactor 🤙